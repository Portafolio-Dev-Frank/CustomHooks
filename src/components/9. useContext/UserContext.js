import { createContext } from 'react';

// HIGH ORDER COMPONENT 
export const UserContext = createContext(null);

