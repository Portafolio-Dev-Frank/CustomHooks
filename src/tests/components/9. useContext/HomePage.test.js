import { shallow } from "enzyme"
import { HomePage } from "../../../components/9. useContext/HomePage"
import { UserContext } from "../../../components/9. useContext/UserContext"


describe('pruebas en <HomePage></HomePage>', () => {

    const user = {
        name: 'francisco',
        email: 'francisco.email.co'
    }

    const wrapper = shallow(
        <UserContext.Provider value={{
            user,
        }}>
            <HomePage></HomePage>
        </UserContext.Provider>
    )

    test('debe renderizar el homePage', () => {
        expect(wrapper).toMatchSnapshot();
    })
    

})
