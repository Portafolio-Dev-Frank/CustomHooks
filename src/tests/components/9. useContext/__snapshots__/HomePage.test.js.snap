// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <HomePage></HomePage> debe renderizar el homePage 1`] = `<HomePage />`;
